app/src/main/java/br/com/cantinho/android_mvvm_clean_architecture/ui/calculator/usecases/SubtractionUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package br.com.cantinho.android_mvvm_clean_architecture.ui.calculator.usecases
+
+import br.com.cantinho.android_mvvm_clean_architecture.ui.calculator.OperationResult
+import br.com.cantinho.android_mvvm_clean_architecture.ui.calculator.Success
+
+class SubtractionUseCase {
+
+    fun performSubtraction(paramA: Int, paramB: Int): OperationResult {
```

Comment:
I think the user case name is already meaningfully. Maybe it's better change method's name `performSubtraction` to perform only.